android-components/components/browser/engine-gecko/build.gradle

```diff
@@ -89,4 +90,20 @@ dependencies {
 apply plugin: "org.mozilla.telemetry.glean-gradle-plugin"
 apply from: '../../../android-lint.gradle'
 apply from: '../../../publish.gradle'
+apply plugin: "org.mozilla.appservices.nimbus-gradle-plugin"
+nimbus {
+    // The path to the Nimbus feature manifest file
+    manifestFile = "geckoview.fml.yaml"
+
+    channels = [
+            debug: "debug",
+            release: "release",
```

Comment:
Should we also have nightly channel?

Reply:
Imported channels are less useful, and more confusing than I thought they were going to be. We may need to add channels here, depending on how the build is setup.